src/hipchatbot.js

```diff
@@ -0,0 +1,198 @@
+import HipchatterBase from 'hipchatter';
+import trebek from './trebek';
+import winston from 'winston';
+import request from 'request';
+import moment from 'moment';
+import App from './models/App';
+import { HIPCHAT_CALLBACK_URL } from './config';
+
+export default class Hipchatter extends HipchatterBase {
+  get_session(callback) {
+    this.request('get', 'oauth/token/'+this.token, callback);
+  }
+}
+
+export default class HipchatBot {
+  constructor(express) {
+    this.express = express;
+    this.start();
+  }
+
+  async start() {
+    this.app = await App.get();
+    if (!this.app.hipchat.oauthId || !this.app.hipchat.oauthSecret) {
+      throw new Error('Before you can run Jeopardy Bot on Hipchat, you must first install it as an add-on.');
+    }
+
+    this.express.get('/capabilities', this.buildCapabilitiesDescriptor.bind(this));
+    this.express.post('/install', this.install.bind(this));
```

Comment:
I would follow the React route here and bind all of these methods in the constructor. Looks cleaner.